guide/migration-from-riot2.md

```diff
@@ -41,12 +41,16 @@ Notice that now `data-is` could be used also in nested tags and it supports dyna
 ### Deprecate `name` and `id` to cache DOM nodes
 
 The attributes `name` and `id` will be no longer used by riot to cache DOM references to the tag context.
-Use the `ref` attribute instead
+Use the `ref` attribute instead.
+children tags are now targeted via `ref` too.
```

Comment:
In my experiment, `ref="subtag"` does not target the node but the riot tag.
![image](https://cloud.githubusercontent.com/assets/2950505/21012602/2237da2e-bd57-11e6-94a1-554106d293db.png)